Examples/restcomm-olympus-nopod/restcomm-olympus/MainTableViewController.mm

```diff
@@ -144,7 +144,7 @@ - (void)viewDidLoad {
     [self.parameters setObject:[NSString stringWithFormat:@"%@", [Utils sipRegistrar]] forKey:@"registrar"];
     
     // initialize RestComm Client by setting up an RCDevice
-    self.device = [[RCDevice alloc] initWithParams:self.parameters delegate:self andICEConfigType:kCustom];
+    self.device = [[RCDevice alloc] initWithParams:self.parameters delegate:self andICEConfigType:kXirsysV2];
```

Comment:
As already discussed in previous review, let's remove andICEConfigType and use self.parameters instead

Reply:
done.